package.json

```diff
@@ -105,8 +113,12 @@
     "rollup-plugin-commonjs": "^9.2.0",
     "rollup-plugin-node-resolve": "^3.4.0",
     "rollup-plugin-size-snapshot": "^0.7.0",
+    "rollup-plugin-typescript": "^1.0.1",
     "storybook-readme": "^5.0.5",
     "tinycolor2": "^1.4.1",
+    "ts-jest": "^24.1.0",
+    "tslib": "^1.10.0",
```

Comment:
It doesn't look like we're using this: https://github.com/Microsoft/tslib#usage

Reply:
Both are dependecies I was told to install in the TS setup for rollup and storybook
`ts-jest`: https://storybook.js.org/docs/configurations/typescript-config/ 
`tslib`:  https://github.com/rollup/rollup-plugin-typescript